app/components/find/courses/international_students_component/view.html.erb

```diff
@@ -20,9 +20,11 @@
 
     <%= t("find.international_candidates.#{visa_type}.#{sponsorship_availability}.html") %>
 
-    <p class="govuk-body">Alternatively, check if you’re eligible for a different type of visa, such as a Graduate or Family visa, that allows you to train to be a teacher without being sponsored.</p>
+    <% if course_has_relocation_entitlement? %>
+      <%= t("find.international_candidates.entitlement.html") %>
+    <% end %>
 
-    <p class="govuk-body">Find out more about <%= govuk_link_to("training to teach in England", "https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen") %>.</p>
+    <p class="govuk-body">Learn more about <%= govuk_link_to("training to teach in England as an international student", "https://getintoteaching.education.gov.uk/non-uk-teachers/train-to-teach-in-england-as-an-international-student") %>.</p>
```

Comment:
Can we keep the links altogether in here? 

https://github.com/DFE-Digital/publish-teacher-training/blob/ce50d4876c7190b333541655fe4444b6714b993f/config/locales/find.yml#L96